### DISCUSSION TITLE
New docs website has been launched!

### DISCUSSION THREAD

**Discussion by rstoenescu** [quasarframework/quasar#15636] [Announcements]
Hello All,

We are thrilled to announce the launch of our new documentation website for Quasar Framework! We've been working hard to create a better, more user-friendly experience for our current users as well as for the ones that are just getting started with our framework.

Please note that this is just a first step. We'll continue to iterating over the site to improve it in the coming months.

We hope that you will all enjoy the new interface, with its dark and light themes (from which you can choose your favorite), and the many other UI improvements (like the API cards as one quick example).

We also want to thank everyone who's been contributing to the new website with feedback. It was thrilling to see the passion in some of the comments :)

And last but not least, **we would like to thank all of our sponsors and backers** for their continued support that allows us to make progress! If you're not one of them and you like what we do, please take a look at our [sponsorship page](https://donate.quasar.dev).

## Important
You'll get the new docs as soon as your currently installed PWA updates to it. If you don't want to wait, uninstall the current instance of it (optionally also clear the browser cache) and refresh the page.

## Donations
Quasar Framework is an open-source MIT-licensed project made possible due to the **generous contributions** by [sponsors and backers](https://github.com/sponsors/rstoenescu).

We've been hit hard because of the recent Github Sponsorship decision to drop Paypal support, so if you are interested in supporting this project, please consider the following:

- [Becoming a sponsor on Github](https://github.com/sponsors/rstoenescu)
- [One-off donation via PayPal](https://paypal.me/rstoenescu1)

**Comment by kurtisas-k** [comment]
Kudos on the redesign. It looks amazing.

**Comment by barisener** [comment]
Looking great 👏🏻

**Comment by Cmacu** [comment]
Looks awesome. Love the dark theme! Thank you!

**Comment by pratik227** [comment]
Looks awesome :rocket::rocket:

**Comment by beliven-emanuele-bertoldi** [comment]
The released version seems a lot better than initial drafts, however I'll still suggest to improve typography removing a lot of styling that make it more hard to read and more cluttered.

I made an experiment removing most of letter-spacing, reducing the general font-weight to 300 and using Montserrat also for doc headings. This is the result and looks to me a little bit cleaner and "professional":

![image](https://user-images.githubusercontent.com/93123952/228457221-8903bdf5-bc1d-4009-a3e9-a0ded9b66819.png)

**Reply by BenJackGill** [reply]
I agree with this adjustment

**Reply by rstoenescu** [reply]
This is a BAD idea. Especially to recommend to others. We are continuously tweaking and improving the docs. At some point, any code on top of the docs will get out of date or will interfere to the way that it should be displayed. I reduced the typography size yesterday so if you still have this on top of the docs you maybe won't even notice it. Moreover, the website could show in a much worse way than we intended. But bottom line: you won't even know if something is wrong from our side or yours.

What we recommend is telling us if something seems out of place so we can fix it for all our developers using the docs.

To those reading this thread: don't overwrite the CSS :)

**Comment by geminigeek** [comment]
hi,

just an observation , i liked the old style , not a fan of new typography and design.  looks better at 90% -80% zoom

**Comment by robert-hoffmann** [comment]
Congrats for the launch

But some small bugs ->
https://quasar.dev/quasar-plugins/dialog

The API part is missing all the documentation on all the options & stuff you can pass to the quasar dialog, like here

```
  $q.dialog({
    component: CustomComponent,

    // props forwarded to your custom component
    componentProps: {
      text: 'something',
      // ...more..props...
    }
  }).onOk(() => {
    console.log('OK')
  }).onCancel(() => {
    console.log('Cancel')
  }).onDismiss(() => {
    console.log('Called on OK or Cancel')
  })
```

It doesn't mention anything about: onOk, onCancel, or how you can add buttons, configure them, etc

IIRR this was all present in the old doc :-)

**Reply by rstoenescu** [reply]
![image](https://user-images.githubusercontent.com/4665641/228573112-331d0838-b79c-45f1-9329-a256b69ec955.png)

Click on the "plus" buttons.

**Reply by robert-hoffmann** [reply]
👍

**Comment by Dimonka2** [comment]
It will be very helpful to have a light version of the site. I'm having a hard time with the website's dark color scheme since I'm used to a lighter background. It's straining my eyes and making it challenging to read.
Additionally, agree with the comment above from @geminigeek about font size.
Otherwise it looks good.

**Reply by IlCallo** [reply]
Landing and components page are dark-theme only, but you can actually change the docs to light theme with the top right icon

![image](https://user-images.githubusercontent.com/10036108/228601311-568b7162-12c0-4796-9e28-d910b1ceffb4.png)

**Reply by Dimonka2** [reply]
Thank you!

**Comment by sluxzer** [comment]
I love this new design, thank you for such great work to all team.

by the way, the animation and concept are fantastic, whose idea is it?  it makes me wonder how you guys implement it and jump to F11 XD. Do you guys make it with some tools to generate the animation and CSS?

**Reply by IlCallo** [reply]
Thanks for the kind words :)
[Dreamonkey](https://dreamonkey.com/), and especially @NiccoloMenozzi, worked on the concept for the renewed Quasar branding and designed the new landing and components page
We worked side-to-side with the core team to add a bit more "space flavor" to Quasar website 👾 🌟

**Reply by IlCallo** [reply]
The stars animation is actually 3 CSS animations one over the other, in an infinite loop
Each layer has stars of different size and moves at a different speed

**Reply by rstoenescu** [reply]
Remember that it's not just the design. The inner workings are also 10 times better ;)

**Comment by it-xtech-dev** [comment]
Congrats to all team members!

I would also vote on smaller (80-90%) fonts for desktop (my is 23'' 1920 x 1080) an its bit hard to read. For much larger screens it might be fine. Mobile font sizes are ok.

**Comment by mariokresic** [comment]
Looks much better!

**Comment by sainf** [comment]
Some notes.
- Typography (Size too big, and sorry, the font is ugly)
- Add an option to hide the right tab to have more space for api boxes.

I liked the previous one better, but maybe I spent too too long on the previous docs.

**Reply by rstoenescu** [reply]
I've adjusted the typography level yesterday and redeployed. It should be fine now.

**Reply by sainf** [reply]
Thanks, It looks a lot better!

Is there any change to make the button to hide the left menu up to 1600px.
In 24" (1920px) it looks fine, but in smallers 19" (1440px and 1600px) the main page it seems to tight.

**Reply by Cmacu** [reply]
> Is there any change to make the button to hide the left menu up to 1600px. In 24" (1920px) it looks fine, but in smallers 19" (1440px and 1600px) the main page it seems to tight.

Vote against this. I use it at half screen as PWA and having the left menu available is valuable. Also I find the site to work/look best at ~75% zoom and this leaves good amount of space for the main content.

**Reply by sainf** [reply]
This is a promenor between 1300px and 1600px.
It is more readable in API Explorer than on the Component page. I'm not asking to change the logic of the PWA, but an option / button to retract one of the navigation blocks to give more space would be welcome.

**Comment by JokerMartini** [comment]
I agree the design is nice but the typography is huge and the bold makes it look clunky and out dated.

**Reply by JokerMartini** [reply]
I find the site looks best when changing the overall zoom level to 70%

**Reply by rstoenescu** [reply]
I've adjusted the typography level yesterday and redeployed. It should be fine now.

**Reply by JokerMartini** [reply]
great work @rstoenescu overall it's a nice improvement.

**Comment by dinyo** [comment]
sorry, i prefer previous version. i think its taking too much space vertically.

**Comment by fprijate** [comment]
Nice design.
Maybe site should remember (by cookie) decision about dark or light mode.

**Reply by IlCallo** [reply]
It does
![image](https://user-images.githubusercontent.com/10036108/229453663-16c56c0a-e5c6-4808-8372-ecefcdd74c61.png)

**Comment by fogx** [comment]
Looks great. I had some troubles with the search though. What's the right place to suggest doc search improvements, just create a regular issue?

**Reply by IlCallo** [reply]
You can write it here if you wish

**Reply by fogx** [reply]
Sure. I found documentation on BEX, but didn't know what that is. The page exists, but doesn't rank highly in the docs search (i found it via google).
<img width="142" alt="Untitled" src="https://user-images.githubusercontent.com/10303647/229465350-88cd38fa-0739-4ad0-8d33-2a97bbb5516c.png">



I saw the doc search uses meilisearch, but couldn't find any public configuration of that.

**Reply by IlCallo** [reply]
Unluckily I think we don't have that much granular power about the indexing and research of contents, but we'll see if we can fix it in some way

**Reply by rstoenescu** [reply]
The indexing is as custom as possible, but not granular enough. However, we have tags set for each page. But remember this is not ChatGPT, so searching for "browser extension" or "bex" is better than a natural language request like "what is a bex".
In any way, we'll see how we can further tweak the tags.

**Comment by topvis** [comment]
1. The link "Electron specific packages" on page https://quasar.dev/quasar-cli-vite/developing-electron-apps/electron-packages is not valid any more.
2. The subject title Developing Mobile Apps (under Quasar CLI (with Vite and Webpack)) : I would suggest to change it to "Developing _Cross-platform_ Apps" since Electron JS can also be used to create Windows desktop apps.

**Reply by IlCallo** [reply]
The page you linked works for me, can you retry?

**Reply by rstoenescu** [reply]
Developing Mobile Apps refers to Cordova and Capacitor --- apps on mobile phones/tablets for delivery with App Store / Play Store.

**Reply by topvis** [reply]
> The page you linked works for me, can you retry?

The page https://quasar.dev/quasar-cli-vite/developing-electron-apps/electron-packages is OK. The link on that page is not valid. Please refer to the screenshot below:
![image](https://user-images.githubusercontent.com/5996531/229604239-d41ca740-9bd9-4a30-8334-30dc95965f9b.png)

**Reply by IlCallo** [reply]
Ok, that's clearer now, thanks

**Reply by rstoenescu** [reply]
Fixed it (the electron link)

**Comment by BenJackGill** [comment]
The typography was tweaked a few days ago, and it looks much better now.

But the code examples still have the old large font.

I suggest reducing them also to match everything else.

I am talking about these parts I highlighted in red:

<img width="1425" alt="Screenshot 2023-04-07 at 12 57 55 pm" src="https://user-images.githubusercontent.com/2043257/230550599-1e4e6828-ddc2-40db-b048-593e004e30ec.png">

**Reply by rstoenescu** [reply]
Tweaked it and redeployed the docs.

**Comment by sam-leishman** [comment]
The new site looks great! I do have a suggestion about the landing and components pages, though.

I feel like the white cards don't work too well with the dark space background. Seems like too much of a contrast for a page that has a dark mode look/feel. I think it would look better if the cards behaved like they were in dark mode.

<img width="1717" alt="Screen Shot 2023-04-10 at 1 02 47 PM" src="https://user-images.githubusercontent.com/70149059/230981129-8c43f851-1306-4020-be0b-5ad066c59588.png">

**Reply by IAdalarI** [reply]
I agree

**Reply by IlCallo** [reply]
Those cards are **meant** be to high contrast

**Comment by enpasos** [comment]
Incredibly appealing! Congratulations!

A question about the making of ...

When I open [https://quasar.dev/why-donate](https://quasar.dev/why-donate) and scroll down, I see _"Ready for more?"_.

When I naively 
```
git clone https://github.com/quasarframework/quasar.git
cd quasar
cd docs
yarn install
quasar dev
```
open [http://localhost:9090/why-donate](http://localhost:9090/why-donate) and scroll down I don't see  _"Ready for more?"_.

**What did I miss?**

**Reply by enpasos** [reply]
Oh ... I realised that I was missing at least two things... to weave in the _"Ready for more?"_ nav info

1. I should do a build.
2. I need to find out how to do the build for a SPA.

A hint for 2?

**Comment by enpasos** [comment]
... and for _separation of concerns_ I think I should leave my **congratulations for the really amazing work** here and open a new thread for my question ... see [https://github.com/quasarframework/quasar/discussions/15886](https://github.com/quasarframework/quasar/discussions/15886)
